frontend/src/features/permits/components/permit-list/Columns.tsx

```diff
@@ -8,6 +8,7 @@ import { CustomActionLink } from "../../../../common/components/links/CustomActi
 import { getDefaultRequiredVal } from "../../../../common/helpers/util";
 import { getPermitTypeName } from "../../types/PermitType";
 import { Box, Tooltip } from "@mui/material";
+import { useNavigate } from 'react-router-dom';
```

Comment:
Remove this import

Reply:
For "PermitsColumnDefinition", use instead:
```
export const PermitsColumnDefinition = (onDocumentDownloadFail: () => void): MRT_ColumnDef<PermitListItem>[] => [
everything else here...
];
```

Reply:
And you can remove "PermitsNotFoundColumnDefinition" at the bottom, as it's not used

Reply:
In "frontend/src/features/permits/components/permit-list/BasePermitList.tsx", replace ```columns: PermitsColumnDefinition``` with ```columns: PermitsColumnDefinition(() => navigate(ERROR_ROUTES.DOCUMENT_UNAVAILABLE)),```